README.md

```diff
@@ -249,18 +266,36 @@ prevent concurrent, unsafe writes to the same file. Our current approach is
 therefore quite safe yet restrictive. We may revisit this decision based on
 developer feedback.
 
+### Quota
+
+Storage Foundation achieves fast and predictable performance by implementing its
+own quota management system.
+
+The web application must explicitly ask for capacity before storing any new
+data. This request will be granted according to the browser's quota
+guidelines. The application can then use the capacity to execute write
+operations. The application can also delete or overwrite data to make space for
+new data without issuing a new request.
+
+This means that anytime an application starts a new Javascript execution context
+(e.g., new tab, new worker, reloading the page), it must make sure it owns
+sufficient capacity before writing new data. It must therefore either
+delete/shrink old files or request additional capacity.
+
 ## Trying It Out
 
 A prototype of Storage Foundation API is available in Chrome Canary. To enable
-it, launch Chrome with the `-enable-blink-features=NativeIO `flag or enable
-“Experimental Web Platform feature” in ["chrome://flags"](chrome://flags).
+it, launch Chrome with the `-enable-blink-features=StorageFoundation `flag or
+enable “Experimental Web Platform feature” in
+["chrome://flags"](chrome://flags).
```

Comment:
Could you also add a sentence pointing people to our Origin Trial registration page?

i.e. https://developer.chrome.com/origintrials/#/view_trial/1673087261568139265

Reply:
Done.